CREATE_YOUR_FIRST_EXTENSION.md

```diff
@@ -164,28 +164,28 @@ import org.opensearch.sdk.rest.BaseExtensionRestHandler;
 public class CrudAction extends BaseExtensionRestHandler {
 
     @Override
-    protected List<RouteHandler> routeHandlers() {
+    protected List<NamedRoute> routes() {
```

Comment:
This needs to be `public` because of the public method it's inheriting.  Don't hold up PR approval for this, I'll address this in a follow-up doc change.

Reply:
Done.